pages-sections/about-us/constants.js

```diff
@@ -0,0 +1,31 @@
+import FaceJCole from "../../assets/img/faces/jeff-cole.jpg";
```

Comment:
Keeping all these constants in a separate file makes the components much cleaner and easier to follow.